src/iuse.cpp

```diff
@@ -141,14 +141,12 @@ void remove_double_ammo_mod( item &it, player &p )
     }
     p.add_msg_if_player( _( "You remove the double battery capacity mod of your %s!" ),
                          it.tname().c_str() );
-    item mod( "battery_compartment", calendar::turn );
+    item mod( "battery_compartment" );
     p.i_add_or_drop( mod, 1 );
```

Comment:
`Character::i_add_or_drop` should be changed to take an item by value, the non-const reference is not needed at all.